includes/wikia/template/PHPEngine.class.php

```diff
@@ -54,6 +56,11 @@ public function render( $template ) {
 		wfProfileOut( __METHOD__ . " - template: {$this->path}" );
 
 		$contents = ob_get_clean();
+		if( $contents == '' ){
+			WikiaLogger::instance()->warning( "Template parse returned empty string: \"{$this->path}\"" , [
```

Comment:
Would be easier if the log message would not be parametrized. We have path as a log context anyway.